src/linter/block_linter.go

```diff
@@ -598,11 +598,15 @@ func (b *blockLinter) checkArray(arr *ir.ArrayExpr) {
 	haveImplicitKeys := false
 	keys := make(map[string]ir.Node, len(items))
 
-	for _, item := range items {
+	multiline := arr.Position.EndLine-arr.Position.StartLine != 0
+
+	for index, item := range items {
 		if item.Val == nil {
 			continue
 		}
 
+		b.checkMultilineArrayTrailingComma(multiline, index, items)
```

Comment:
I don't exactly like passing a bool and then return inside a function based on that argument.

It's better to check before calling a function or to pass `arr` there and check for the multiline inside it.
`multiline` variable is not needed inside checkArray function anyway.

Reply:
Done